docs/content/specs/terraform/_index.md

```diff
@@ -555,7 +555,7 @@ var.new_network_security_group_name == null ? "${var.subnet_name}-nsg" : var.new
 
 <br>
 
-#### ID: TFFR14 - Category: Inputs - No `enabled` or `module_depends_on` variable
+#### ID: TFNFR14 - Category: Inputs - No `enabled` or `module_depends_on` variable
```

Comment:
Hey @jtracey93, the fix is correct - but can we just change it, or is that value referenced anywhere else?

Reply:
@AlexanderSehr, after doing a quick cross-repo search: it's referenced in the compoisition.md file, but @MariusStorhaug has also updated the reference, so we're good. 👍

Reply:
The real question is if this counts as a functional requirement (in which case the TFFR ID is correct, but then the entry should be put under the Functional requirements chapter), or a as a non-functional requirement (in which case it should be renamed to TFNFR as Marius is suggesting. I'd leave this part of the question for @jtracey93 to decide. :)

Reply:
@matebarabas: Suggest you search for '&' and make a call on what is correct. Assuming the official AVM do should be more formal than spoken informal language, there are some use of [contractions](https://en.wikipedia.org/wiki/Contraction_(grammar)) still.

Reply:
IMHO it's a functional requirement as it's about usability of the module via its inputs

Reply:
Should it then be under the other list as well?

Reply:
So, it should then be changed back to TFFR14 and shown under the related section.

Reply:
Reverted updating numbers after talking with @matebarabas ;)